File: Trivia/GetPlayersInRoomResponse.cpp
#include "GetPlayersInRoomResponse.h"

GetPlayersInRoomResponse::GetPlayersInRoomResponse(vector<string> _players, const unsigned int& maxPlayers) : players(_players), max_players(maxPlayers)
{
}

vector<string> GetPlayersInRoomResponse::getPlayers()
{
	return players;
}

unsigned int GetPlayersInRoomResponse::getMaxPlayers()
{
	return max_players;
}

File: TriviaClient/Responses/GetPlayersInRoomResponse.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace TriviaClient.Responses
{
    public class GetPlayersInRoomResponse
    {
        private string[] players;
        private int max_players;
        public GetPlayersInRoomResponse(string[] players, int max_players)
        {
            this.players = players;
            this.max_players = max_players;
        }

        public string[] GetPlayers()
        {
            return players;
        }

        public int GetMaxPlayers()
        {
            return max_players;
        }

    }
}
